bread/views/read.py

```diff
@@ -49,9 +49,11 @@ def layoutasreadonly(layout):
         and (
             isinstance(
                 element,
-                (_layout.form.InlineDeleteButton, _layout.form.FormsetAddButton),
+                hg.BUTTON,
             )
             or getattr(element, "attributes", {}).get("type") == "submit"
         )
     )
+    for form in layout.filter(lambda element, ancestors: isinstance(element, hg.FORM)):
+        form.attributes["onsubmit"] = "return false;"
```

Comment:
This is a small improvment to prevent changes beeing submitted "accidentally" on the read-view. Browser seem to submit a form when any button (or the first button) is clicked inside the form if no submit button has been defined. This is a bit problematic because e.g. the carbon tags use a button element for representation.